sweepai/utils/openai_proxy.py

```diff
@@ -0,0 +1,37 @@
+import os
+import openai
+from sweepai.config.server import OPENAI_API_TYPE, OPENAI_API_ENGINE
+
+class OpenAIProxy:
+    def __init__(self, OPENAI_API_KEY_GPT35, OPENAI_API_KEY_GPT4, OPENAI_API_KEY_GPT4_32K, OPENAI_FALLBACK):
+        self.OPENAI_API_KEY_GPT35 = OPENAI_API_KEY_GPT35
+        self.OPENAI_API_KEY_GPT4 = OPENAI_API_KEY_GPT4
+        self.OPENAI_API_KEY_GPT4_32K = OPENAI_API_KEY_GPT4_32K
+        self.OPENAI_FALLBACK = OPENAI_FALLBACK
+
+    def call_openai(self, model, **kwargs):
```

Comment:
Make sure these are set, you can fetch these from server.py
    openai.api_type = OPENAI_API_TYPE
    openai.api_base = OPENAI_API_BASE
    openai.api_version = OPENAI_API_VERSION